iib/workers/tasks/iib_static_types.py

```diff
@@ -39,6 +39,7 @@ class PrebuildInfo(TypedDict):
     binary_image: str
     binary_image_resolved: str
     bundle_mapping: NotRequired[Dict[str, List[str]]]
+    bundle_replacements: NotRequired[Dict[str, str]]
```

Comment:
Changes in this file are not required since we are not making any changes to the worker yet.

Reply:
This comment is not valid anymore. Resolving the thread